src/reducers/index.js

```diff
@@ -89,6 +89,7 @@ const combinedReducer = combineReducers({
   // These props don't have any actions associated with them
   display: createReducer(null),
   dataSource: createReducer(null),
+  modeOptions: createReducer({}),
```

Comment:
[Nit] modeOptions inside options seems a bit confusing to me. Was there any consideration of having reFocus inside display ? If not, was there any reason to separate this ? 
What does mode mean here ?

Reply:
if we expand on how flowchart behaves, something like `behavior` or something along those lines might give a better idea of what the key value is. For me `modeOptions` does not seem fitting

Reply:
@ravi-kumar-pilla 
Please refer https://github.com/kedro-org/vscode-kedro/issues/126#issuecomment-2435157272

Reply:
hey since the stand-alone version primarily covers the flowchart, we might not need to specify 'flowchart.' Maybe something more general like `behaviour`, `interaction`, or `interactionMode` would work?